public/components/Commons/Constants.tsx

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+export const DEFAULT_MESSAGE_SOURCE = {
```

Comment:
Let's move this to public/utils/constants.ts

Reply:
done